torch/csrc/jit/profiling_graph_executor_impl.cpp

```diff
@@ -149,6 +149,13 @@ ExecutionPlan ProfilingGraphExecutorImpl::getPlanFor(Stack& stack) {
   return *optimized_plan_;
 }
 
+std::shared_ptr<Graph> ProfilingGraphExecutorImpl::_getProfiledGraph() const {
+  if (!pr_ || !optimized_plan_) {
```

Comment:
Hi @Krovatkin , thanks for providing this feature! I have been trying out this branch to get the profiled graph. But so far I'm only getting nullptr from this api. The `optimized_plan_` is always nullptr. I'm calling the api with the below snippet, there must be something I'm missing.
```c++
  m.def(
      "_jit_get_profiled_graph",
      [](std::shared_ptr<Graph> graph, const std::vector<at::Tensor>& inputs) {
        auto executor = GraphExecutor(graph);
        Stack stack(fmap<IValue>(inputs));
        executor.getPlanFor(stack);
        return executor.getProfiledGraph();
      });
```

Reply:
@BowenBao could try running it twice  i.e. `executor.run(stack); executor.run(stack);` and then do `executor.getPlanFor()`. We definitely need to run it at least once, so we collect shapes information. Now that I'm looking at this API, I don't think we necessarily need to give you the optimized graph (i.e. making you run the executor twice). I'll push a patch so you can get the profiled graph after you run the executor once.

Reply:
@BowenBao arggh, you are right this is wrong! I didn't really think python API through. let me push a patch to fix this!